engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -355,4 +366,91 @@ private static boolean isErrorLevelEnable()
     return false;
   }
 
+  /**
+   * Adds Logger Appender
+   * @param name Appender name
+   * @param refName Appender reference name (appender name in the properties)
+   * @param properties Appender properties
+   */
+  public static void addAppender(String name, String refName, Properties properties)
+  {
+    if (getAppendersNames().contains(name)) {
+      logger.warn("A logger appender with the name '{}' exists. Cannot add a new logger appender with the same name", name);
+    } else {
+      try {
+        Method method = PropertyConfigurator.class.getDeclaredMethod("parseAppender", Properties.class, String.class);
+        method.setAccessible(true);
+        Appender appender = (Appender)method.invoke(new PropertyConfigurator(), properties, refName);
+        if (appender == null) {
+          logger.warn("Cannot add a new logger appender. Name: {}, Reference name: {}, Properties: {}", name, refName, properties);
+        } else {
+          appender.setName(name);
+          Logger.getRootLogger().addAppender(appender);
+        }
+      } catch (Exception ex) {
+        logger.warn("Cannot add a new logger appender. Name: {}, Reference name: {}, Properties: {}", name, refName, properties, ex);
+      }
+    }
+  }
+
+  /**
+   * Adds Logger Appenders
+   * @param namesAndRefs Name and reference name in the logger properties
+   * @param args Args with properties
+   * @param propertySeparator Property separator
+   */
+  public static void addAppenders(List<Pair<String, String>> namesAndRefs, String args, String propertySeparator)
+  {
+    try {
+      Properties properties = new Properties();
+      properties.load(new StringReader(args.replaceAll(propertySeparator, "\n")));
+      for (Pair<String, String> nameAndRef : namesAndRefs) {
+        addAppender(nameAndRef.getFirst(), nameAndRef.getSecond(), properties);
+      }
+    } catch (IOException ex) {
+      ;
+    }
+  }
+
+  /**
+   * Adds Default Logger Appender
+   * Syntax of a value of the default appender parameters: {reference-name},{string-with-properties}
+   * Comma is a separator between properties
+   * @param name Appender name
+   */
+  public static void addAppender(String name)
+  {
+    String appenderParameters = System.getProperty(LogicalPlan.LOGGER_APPENDER.getLongName());
```

Comment:
You could potentially support multiple appenders here. Consider renaming the attribute to LOGGER_APPENDERS.